person.rb

```diff
@@ -0,0 +1,33 @@
+require './corrector'
+
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(9999)
+    @parent_permission = parent_permission
+    @age = age
+    @name = name
+    @corrector = Corrector.new
+    @rentals = []
+  end
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] in order to make this work you will need to initialize the rental class in this method as explained in [this article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`).